index.html

```diff
@@ -63,5 +63,143 @@ <h2 class="let">LET'S CONNECT</h2>
       
     </div>
     </section>
+    <section class="works">
+      <div class="card">
+        <div class="card__image">
+          <img src="images/tox1.png" alt="">
```

Comment:
- [ ] Speaking of specifying paths for images, here is another bug that is preventing your beautiful images to be displayed on your page 🤔. To fix this kindly capitalize the first letter of the `images` directory, so you end up with something like this `src="Images/tox1.png"`

![image](https://user-images.githubusercontent.com/43172164/206504012-413e61cf-9bef-456b-ab8b-0ff5d625d5aa.png)